### ISSUE TITLE
How to know the order of classes for multiclass problem when using predict_proba?

### ISSUE THREAD

**Issue by juliuskittler** [mljar/mljar-supervised#400]
Assume I have a multiclass classification problem where my target `y` in the training data is a 1D-vector with strings for the labels. In the example below, the labels can be `['Fair', 'Good', 'Ideal', 'Premium', 'Very Good']`. 

After fitting a multiclass model given this `y`, I want to use the `predict_proba` function. This function gives me a NumPy array with shape (n_rows, 5) because there are 5 classes. The problem is that I don't know which level of the second dimension corresponds to which class. 

**Question:** 
How do I find out which level of the second dimension corresponds to which class? 

Maybe it would be better to return a data frame with columns representing class labels here? Or to let the user specify the order of class labels somehow? Or to force the user to provide the target in a (n_rows, 5) format after one-hot encoding?

**Example:** 
```shell
import pandas as pd
from supervised import AutoML

# Import data
url = "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/diamonds.csv"
df = pd.read_csv(url)
display(df.head())

# Split in train and target
x = df.drop(columns = ["cut"])
y = df.cut.to_numpy()

# Fit model
model = AutoML(mode="Perform",
            eval_metric="logloss",
            explain_level=0,
            total_time_limit=60,
            results_path=None,
            ml_task = "multiclass_classification")
model.fit(x, y)

# Predict probabilities for training data
pred = model.predict_proba(x)
print(pred.shape)
print(pred)
```

**Comment by pplonski** [comment]
@juliuskittler great question - sorry that you cant find answer in the docs.

As you suggested, there is `predict_all()` method that returns DataFrame with columns with class names.

I will update docs.

**Comment by pplonski** [comment]
Added infomation in Readme in FAQ. Thank you @juliuskittler!
